app/controllers/provider_interface/provider_users_controller.rb

```diff
@@ -82,6 +82,21 @@ def update_providers
       redirect_to provider_interface_provider_user_path(provider_user)
     end
 
+    def confirm_remove
+      @provider_user = ProviderUser.visible_to(current_provider_user).find(params[:provider_user_id])
```

Comment:
This can use `find_provider_user` from https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2013 now.